src/common/snippets/src/op/perf_count.cpp

```diff
@@ -62,9 +164,27 @@ void PerfCountBegin::set_start_time() {
 }
 
 //////////////////PerfCountEnd///////////////
-PerfCountEnd::PerfCountEnd(const Output<Node>& pc_begin) : PerfCountEndBase({pc_begin}), accumulation(0ul), iteration(0u) {
+
+PerfCountEnd::PerfCountEnd() : PerfCountEndBase() {}
+
+PerfCountEnd::PerfCountEnd(const Output<Node>& pc_begin,
+                           std::vector<std::shared_ptr<utils::Dumper>> dumpers,
+                           const std::string& params)
+    : PerfCountEndBase({pc_begin}),
+      accumulation(0ul),
+      iteration(0u),
+      dumpers(dumpers) {
```

Comment:
Please avoid redundant copy, since you are passing by value
```suggestion
      dumpers(std::move(dumpers)) {
```

Reply:
Done